### PR TITLE
Advancing 0.15.0-rc1 release to status: admiral

### PR DESCRIPTION
**Pull request by dfarrell07**
Depends on https://github.com/submariner-io/submariner-operator/pull/2605

### CODE REVIEW

**Review by tpantelis — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

**Comment by dfarrell07** [comment]
The operator release job isn't finished https://github.com/submariner-io/submariner-operator/actions/runs/4787228603

File: releases/v0.15.0-rc1.yaml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 version: v0.15.0-rc1
 name: 0.15.0-rc1
 branch: release-0.15
-status: shipyard
+status: admiral
 components:
   shipyard: 3236ac72011806d4a855dc7847c67810b2c13f33
+  admiral: e4c1d20bc64a07eaff7312428d63e8df02441d00
```
Comment (dfarrell07) [inline releases/v0.15.0-rc1.yaml:8]: https://github.com/submariner-io/admiral/commit/e4c1d20bc64a07eaff7312428d63e8df02441d00 :heavy_check_mark:

**Comment by dfarrell07** [comment]
> The operator release job isn't finished https://github.com/submariner-io/submariner-operator/actions/runs/4787228603

Maybe that isn't an issue in this case? I just think it's a good practice to wait on the release jobs between steps.

**Comment by dfarrell07** [comment]
> > The operator release job isn't finished https://github.com/submariner-io/submariner-operator/actions/runs/4787228603
> 
> Maybe that isn't an issue in this case? I just think it's a good practice to wait on the release jobs between steps.

This is finished, editing the OP to get Depends on bot to re-check.

**Comment by dfarrell07** [comment]
https://github.com/submariner-io/releases/actions/runs/4787491149

**Pull request merged**